crates/web5/src/credentials/mod.rs

```diff
@@ -1,4 +1,5 @@
 mod create;
+pub mod credential_schema;
```

Comment:
`use credentials::credential_schema::CredentialSchema` vs `use credentials::CredentialSchema`. up to you, i prefer the latter.

Reply:
nice yeah totally agree